File: tests/fixtures/repomap/sample.dart
import 'dart:async';
import 'package:http/http.dart' as http;

class UserService {
  final Database db;

  UserService(this.db);

  Future<User?> getUser(int id) async {
    return db.find(id);
  }

  Future<List<User>> listUsers() async {
    return db.all();
  }
}

enum UserRole {
  admin,
  user,
  guest,
}

void createApp(Map<String, dynamic> config) {
  // setup
}

String _internalHelper() {
  return 'helper';
}

File: tests/fixtures/repomap/sample.rb
require 'json'
require_relative 'helpers'

class UserService
  def initialize(db)
    @db = db
  end

  def get_user(id)
    @db.find(id)
  end

  def list_users
    @db.all
  end
end

module Validators
  def self.validate_email(email)
    email.include?('@')
  end
end

def create_app(config)
  UserService.new(config[:db])
end
